app/src/androidTest/java/ch/epfl/sdp/blindly/PermissionTest.kt

```diff
@@ -40,12 +46,10 @@ class PermissionTest {
 
     @Test
     fun testLocationPermissionFiresProfileName() {
-        activityRule.scenario.onActivity { activity ->
```

Comment:
I don't know if you were able to fix this (and thus removed this part) but sometimes there's a system dialog that pops up atop of the activity, and so cirrus cannot click on the button underneath, which at least used to make the tests fail?

Reply:
Yes, I use the GrantPermissionRule in order to tell the test runner to automatically give permission, so it doesn't pop up.